Microsoft.Toolkit.Uwp/Helpers/CameraHelper/CameraHelper.cs

```diff
@@ -125,7 +130,6 @@ public class CameraHelper : IDisposable
             var settings = new MediaCaptureInitializationSettings()
             {
                 SourceGroup = _group,
-                SharingMode = MediaCaptureSharingMode.SharedReadOnly,
```

Comment:
Why is this removed? Is this the default option for MediaCaptureInitializationSettings and doesn't need to be set explicitly?

Reply:
you need to be able to change the format on the frameSource, SharedReadOnly does not allow that to happen